Cargo.toml

```diff
@@ -34,16 +34,20 @@ shred-derive = "0.6.0"
 tuple_utils = "0.3"
 rayon = { version = "1.0.0", optional = true }
 nonzero_signed = "1.0.1"
+uuid = { version = "0.7.4", optional = true, features = ["v4", "serde"] }
 
 serde = { version = "1.0", optional = true, features = ["serde_derive"] }
 
 [features]
 default = ["parallel"]
 parallel = ["rayon", "shred/parallel", "hibitset/parallel"]
 nightly = ["shred/nightly"]
+uuid_entity = ["uuid", "serde"]
+stdweb = ["uuid/stdweb"]
+wasm-bindgen = ["uuid/wasm-bindgen"]
```

Comment:
Is this something we have to expose?

Reply:
if you want to support both stdweb (cargo-web) and wasm then yes.